lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -14,19 +14,221 @@
 
 package org.partiql.lang.syntax
 
-import com.amazon.ion.*
-import org.partiql.lang.ast.*
+import com.amazon.ion.IonBool
```

Comment:
Yes, it will cause conflicts.  The feedback we've been getting is that star imports are bad and nobody likes them and especially nobody likes having to configure their IDE just for this project.  The style has now "officially" [changed](https://github.com/partiql/partiql-lang-kotlin/commit/f7d32c2e8d8bc05bea5b9840b40d233945619d3f), but I haven't gone so far as to remove the `.*` imports in all files.